langs/vlang.yaml

```diff
@@ -0,0 +1,29 @@
+id: "v"
+aliases:
+  - "vlang"
+name: "V"
+
+install:
+  manual: |
+    git clone https://github.com/vlang/v
+    pushd v
+    make
+    popd
+
+main: "main.v"
+template: |
+  fn main() {
+    println('Hello, world!')
+  }
+
+compile: |
+  v run main.v
+
+format:
+  run: |
+    v fmt -w main.v
```

Comment:
We'll want the formatter to write to stdout rather than writing in place (I'm assuming `-w` is for in-place writing).